site/content/docs/5.2/components/badge.md

```diff
@@ -14,6 +14,11 @@ Badges scale to match the size of the immediate parent element by using relative
 
 ### Headings
 
+<!-- Boosted mod : design callout -->
```

Comment:
IMO you can remove all `<!-- Boosted mod : design callout -->` since `{{< ods-incompatibility-alert >}}` already kind of already doing the job.

Reply:
We collectively chose to keep them. Resolving the issue.